src/lake/Lake/Build/Library.lean

```diff
@@ -73,6 +73,18 @@ def LeanLib.staticFacetConfig : LibraryFacetConfig staticFacet :=
 def LeanLib.staticExportFacetConfig : LibraryFacetConfig staticExportFacet :=
   mkFacetJobConfig (LeanLib.recBuildStatic · true)
 
+@[specialize] protected def LeanLib.buildStaticFat
+(self : LeanLib) : FetchM (BuildJob FilePath) := do
+    withRegisterJob s!"{self.name}:static.fat" do
+    let mods ← self.rootModules.concatMapM fun mod => do
+      return (← mod.transImports.fetch).push mod
```

Comment:
```suggestion
    let mods ← self.modules.fetch
    let mods ← 
      try 
        mods.concatMapM fun mod => do
          return (← mod.transImports.fetch).push mod
      catch errPos =>
        return Job.error (← takeLogFrom errPos)
```

Here is a possible adaption incorporating the changes I suggested regarding `self.modules.fetch` and the `try .. catch` block. (I have not type checked this, so there may be a typo.)

Reply:
```suggestion
    let mods ← (← self.modules.fetch).concatMapM fun mod => do
      return (← mod.transImports.fetch).push mod
```


Due to some touchups on master, the `try ... catch` block is no longer needed.